xml/obs/openSUSE:Leap:15.xml

```diff
@@ -1,12 +0,0 @@
-<openQA
```

Comment:
This file looks generic for all Leap 15.* versions to me. Sure this also needs to be removed?

Reply:
Same comment for the next files as well, I think the `Leap:15:` files need to stay?

Reply:
I noticed that too

Reply:
`project_pattern="openSUSE:Leap:(?P&lt;version&gt;15.[3-9]):ToTest"`
restored the `:15:` files

Reply:
Thank you!